dwave/cloud/config/models.py

```diff
@@ -99,6 +101,12 @@ class PollingSchedule(BaseModel, GetterMixin):
     backoff_base: Optional[float] = 1.3
 
 
+def _literal_eval(obj):
+    if isinstance(obj, str):
+        return ast.literal_eval(obj)
```

Comment:
Didn't know about this, neat

Reply:
Yup, we use it to safely parse linear/quad list/dict in [`dwave sample`](https://github.com/dwavesystems/dwave-cloud-client/blob/68078b1ca19f5534d36b469ac968eaf1c6e6322f/dwave/cloud/cli.py#L606) as well.